cspell.json

```diff
@@ -16,6 +16,6 @@
     "livestream",
     "clonedeep"
   ],
-  "ignoreRegExpList": ["/.*@\\d{13}-/"],
+  "ignoreRegExpList": ["/.*@\\d{13}-/", "/\\d{14}-\\d{3}@.*/"],
```

Comment:
Do we still need the previous ignore regex? Also, could you add a comment detailing it will ignore lexicos? :) It'll make it easier at a glance to know what the regex is doing

Reply:
Done and done!